backend/src/main/java/de/bund/digitalservice/ris/norms/application/exception/StatusNotFoundException.java

```diff
@@ -0,0 +1,33 @@
+package de.bund.digitalservice.ris.norms.application.exception;
+
+import de.bund.digitalservice.ris.norms.utils.exceptions.NormsAppException;
+import java.net.URI;
+import java.util.Map;
+import java.util.UUID;
+import lombok.Getter;
+
+/** Indicates that the requested status does not exist. */
+@Getter
+public class StatusNotFoundException extends RuntimeException implements NormsAppException {
```

Comment:
The naming of the exception is not that matching right? Shouldn't it be something like `ImportProcessNotFoundException`?